### ISSUE TITLE
bigquery, spanner: dependency upgrades broken.

### ISSUE THREAD

**Issue by tritone** [googleapis/google-cloud-go#11344]
## Client

storage, spanner, and bigtable

## Environment

any

## Code and Dependencies

```
go get -u cloud.google.com/go/storage
```

## Summary

Updating dependencies via go get fails with the following:

```
$ go get -u cloud.google.com/go/storage
go: cloud.google.com/go/storage imports
        google.golang.org/grpc/xds/googledirectpath imports
        google.golang.org/grpc/internal/xds/bootstrap imports
        github.com/envoyproxy/go-control-plane/envoy/config/core/v3: ambiguous import: found package github.com/envoyproxy/go-control-plane/envoy/config/core/v3 in multiple modules:
        github.com/envoyproxy/go-control-plane v0.13.1 (/Users/6004148/.local/share/go/pkg/mod/github.com/envoyproxy/go-control-plane@v0.13.1/envoy/config/core/v3)
        github.com/envoyproxy/go-control-plane/envoy v1.32.2 (/Users/6004148/.local/share/go/pkg/mod/github.com/envoyproxy/go-control-plane/envoy@v1.32.2/config/core/v3)
```

Bigtable and spanner modules also have github.com/envoyproxy/go-control-plane as an indirect dependency and so also fail for the same reason.

This was caused by envoyproxy/go-control-plane#714 which split the module into submodules. This was released today in https://github.com/envoyproxy/go-control-plane/releases/tag/envoy%2Fv1.32.2 . An issue has been filed on the repo here: envoyproxy/go-control-plane#1074

## Workaround

The bad release can be excluded from your go.mod as follows:

`go mod edit --exclude=github.com/envoyproxy/go-control-plane@v0.13.2`

This should unblock things for the immediate term.

We will need the issue to be fixed in envoyproxy/go-control-plane, and then we'll do a new release of the affected modules in google-cloud-go in order to fully resolve the issue.

**Comment by valerian-roche** [comment]
Hey, this [PR](https://github.com/envoyproxy/go-control-plane/pull/1075) attempts to solve this issue, but I do not know how to validate whether it will work, given the number of import layers within the google go repositories and my limited understanding of `go get -u` behavior. 
Is anyone with more understanding able to review it?

**Comment by BrennaEpp** [comment]
> Hey, this [PR](https://github.com/envoyproxy/go-control-plane/pull/1075) attempts to solve this issue, but I do not know how to validate whether it will work, given the number of import layers within the google go repositories and my limited understanding of `go get -u` behavior. Is anyone with more understanding able to review it?

Hi @valerian-roche, @codyoss from our team reviewed that PR. We can validate from the Storage side once released

**Comment by valerian-roche** [comment]
> > Hey, this [PR](https://github.com/envoyproxy/go-control-plane/pull/1075) attempts to solve this issue, but I do not know how to validate whether it will work, given the number of import layers within the google go repositories and my limited understanding of `go get -u` behavior. Is anyone with more understanding able to review it?
> 
> Hi @valerian-roche, @codyoss from our team reviewed that PR. We can validate from the Storage side once released

Thanks for the review. Hopefully the latest release (`v0.13.4`) should have addressed the issue

**Comment by tritone** [comment]
This seems to be fixed for storage since we've bumped the dependency on envoy-proxy/go-control-plane and cut a new release of storage (v1.50.0).

Will close this issue once I've confirmed it's fixed for the other libraries.

**Comment by amacneil** [comment]
Also broken in bigquery v1.66.0

**Comment by alvarowolfx** [comment]
@amacneil do you have more details on how to reproduce it with BigQuery ? Are you also using Storage on the same project. I tried to reproduce here with bigquery v1.66.0 and couldn't do it. Just tried with an empty project or upgrade another project here, and both worked just fine. Not sure if I'm missing something to reproduce here on my end.

**Comment by tritone** [comment]
The spanner release also went out yesterday, so I think this issue should be resolved now.

Please open a new issue if you run into any further blockers on updating any of these packages (storage, spanner, bigquery, bigtable).

**Comment by amacneil** [comment]
For others still running into this solution, see https://github.com/googleapis/google-cloud-go/issues/11542#issuecomment-2669493568

Fixed by running:

```
go get -u cloud.google.com/go/storage && go mod tidy
```
